.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
Concerning squashing commits, I am not sure if it makes sense to force contributors to do that. I imagine if there were e.g. 20 commits for a simple change, we could use our judgement and squash it ourselves, but in most cases there probably won't be enough commits to warrant it (though I could be wrong).